C_fibonacci (2).c

```diff
@@ -0,0 +1,21 @@
+
+#include <stdio.h>
+
+int main()
+{
+  int n,n1,n2,n3;
```

Comment:
Change it to long long int. since the program needs to return atleast 100 fibonacci terms.